examples/application/main.go

```diff
@@ -29,10 +30,28 @@ func main() {
 
 	// Use the client with token authentication
 	ctx := context.Background()
+
+	// List applications.
 	apps, err := client.Application.List(ctx, 10, 0)
 	if err != nil {
 		log.Printf("Error listing users: %v", err)
 	} else {
 		fmt.Printf("Found %d applications\n", len(*apps.Applications))
 	}
+
+	// Authorize API.
+	id := "1f616716-f518-48a5-a497-5eb0e2200b4f"
```

Comment:
shall we use a placeholder here, just like "app_uuid" placeholder in L51?

```suggestion
	id := "api_resource_uuid"
```

Reply:
Fixed with https://github.com/asgardeo/go/pull/7/commits/d1b6bd52ab40504cf3374a3430e2088a3d1e5dee